ui/app/src/components/FormsEngine/components/FieldInformationDialog.tsx

```
@@ -0,0 +1,119 @@
+									) : (
+										<Typography>{value ? value : '-'}</Typography>
+									)}
+								</Grid>
+							</Grid>
+						);
+					})}
+				</CardContent>
+			</Card>
+
+			{descriptor.sections?.map((section) => (
+				<Card>
```

Biome (2.3.14)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)